api/src/main/java/com/epam/pipeline/manager/cluster/performancemonitoring/ESMonitoringManager.java

```diff
@@ -249,4 +255,13 @@ private LocalDateTime asMonitoringDateTime(final String dateTimeString) {
         s3.setUsableSpace(s1.getUsableSpace() + s2.getUsableSpace());
         return s3;
     }
+
+    private InputStream convertStatsToCsv(final List<MonitoringStats> monitoringStats) {
+        try {
+            return csvStatsWriter.convertStatsToFile(monitoringStats);
```

Comment:
I suppose we can get rid of checked exceptions being thrown from `CsvMonitoringStatsWriter` to simplify the call site.

Reply:
Removed checked exception from method declaration in the abstract class, process writer-specific exceptions inside a concrete class